README.rst

```diff
@@ -257,7 +257,27 @@ Since going over all (or select) keys in an S3 bucket is a very common operation
   ...     print(key, round(len(content) / 2**20))
   annual/monthly_rain/2010.monthly_rain.nc 13
   annual/monthly_rain/2011.monthly_rain.nc 13
-  annual/monthly_rain/2012.monthly_rain.nc 13
+  annual/monthly_rain/2012.monthly_rain.nc 13 
+
+Specific s3 object version
+--------------------------------------
+
+The ``smart_open``'s ``open`` function has the parameter version_id, which allows you to get the desired version of the file from bucket S3.
+
+*Important*: Version control disabled by default. Before starting to use version_id parameter, enable version control in bucket S3. Read https://docs.aws.amazon.com/AmazonS3/latest/dev/Versioning.html
+
+.. code-block:: python
+
+  >>> from smart_open import open
+  >>> with open('smart-open-bucket', 'test_s3_version.txt', 'rb', version_id='azaMK5Yf_wg03QI1vzKBgzDG8RMin3l1') as file:
```

Comment:
Are you sure this actually works?

version_id is a parameter that gets passed through the transport_params dict, isn't it?